community/samples/serving/helloworld-python-bentoml/README.md

```diff
@@ -0,0 +1,257 @@
+---
+title: "Hello World - Python BentoML"
+linkTitle: "Python Bentoml"
+weight: 1
+type: "docs"
+---
+
+A simple machine learning model with API serving that is written in python and
+using [BentoML](https://github.com/bentoml/BentoML). BentoML is an open source
+framework for high performance ML model serving.
```

Comment:
...ML model serving, which supports all major machine learning frameworks including Keras, Tensorflow, PyTorch, Fast.ai, XGBoost and etc.